setup.py

```diff
@@ -15,7 +15,7 @@
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'cycler',
                       'numpy', 'matplotlib', 'super_state_machine',
-                      'historydict'],
+                      'historydict', 'event_model'],
```

Comment:
Oh, sorry, the above was with my edits to setup.py for `event-model` instead of `event_model`.  Though the result is the same with `event_model` in the install_requires section:

```
(/tmp/foo)edill@edill-810g:~/dev/python/bluesky (remove-schema $ u=)$ psd
running develop
running egg_info
writing top-level names to bluesky.egg-info/top_level.txt
writing dependency_links to bluesky.egg-info/dependency_links.txt
writing requirements to bluesky.egg-info/requires.txt
writing bluesky.egg-info/PKG-INFO
reading manifest file 'bluesky.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'bluesky.egg-info/SOURCES.txt'
running build_ext
Creating /tmp/foo/lib/python3.5/site-packages/bluesky.egg-link (link to .)
bluesky 0.4.3+87.gd35b09f is already the active version in easy-install.pth

Installed /home/edill/dev/python/bluesky
Processing dependencies for bluesky==0.4.3+87.gd35b09f
Searching for event_model
Reading https://pypi.python.org/simple/event_model/
Reading https://pypi.python.org/simple/event-model/
Couldn't find index page for 'event_model' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
^[	No local packages or download links found for event_model
error: Could not find suitable distribution for Requirement.parse('event_model')
```